mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/ElasticSearchIntegrationTestWithDefaultTextExtractor.java

```diff
@@ -19,10 +19,10 @@
 
 package org.apache.james.mailbox.elasticsearch;
 
-    import org.apache.james.mailbox.extractor.TextExtractor;
-    import org.apache.james.mailbox.store.extractor.DefaultTextExtractor;
-    import org.junit.Assume;
-    import org.junit.Test;
```

Comment:
should be a fixup of previous commit

Reply:
That should be fixed when doing last rebase --autosquash before merging